### PR TITLE
Documentation for android service

### PR DESCRIPTION
**Pull request by noeppi-noeppi**
Do not merge until [#138](https://github.com/codeoverflow-org/nodecg-io/pull/138) is merged.

### CODE REVIEW

File: docs/samples/android.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+## Using the Android sample bundle
+
+The Android example bundle in `samples/android` shows how to connect to an android device and turn on WLAN.
+
+1. clone [this](https://github.com/noeppi-noeppi/nodecg-io-android) Git Repository
+
+2. Run `gradlew build` (on windows) or `./gradlew build` (on linux) inside the cloned repository.
```
Comment (daniel0611) [inline docs/samples/android.md:7]: `./gradlew build` throws lint errors for me:
```
> Task :app:lint                                                                                                                                                                                                   
Ran lint on variant release: 37 issues found                                                                                                                                                                       
Ran lint on variant debug: 38 issues found                                                                                                                                                                         
Wrote HTML report to file:///home/daniel/Coding/noeppi-noeppi/nodecg-io-android/app/build/reports/lint-results.html                                                                                                
Wrote XML report to file:///home/daniel/Coding/noeppi-noeppi/nodecg-io-android/app/build/reports/lint-results.xml                                                                                                  

> Task :app:lint FAILED                                                                                  

FAILURE: Build failed with an exception.                                                                 

* What went wrong:                                  
Execution failed for task ':app:lint'.                                                                   
> Lint found errors in the project; aborting build.                                                      
                                                                                                         
  Fix the issues identified by lint, or add the following to your build script to proceed with errors:                                                                                                             
  ...                                               
  android {                                         
      lintOptions {                                 
          abortOnError false                        
      }                                             
  }                                                 
  ...                                               
                                                    
  The first 3 errors (out of 11) were:                                                                   
  /home/daniel/Coding/noeppi-noeppi/nodecg-io-android/app/src/main/java/io/github/noeppi_noeppi/nodecg_io_android/Actions.java:247: Error: Call requires permission which may be rejected by user: code should explicitly check to see if permission is available (with checkPermission) or explicitly handle a potential SecurityException [MissingPermission]
          Location lastKnown = mgr.getLastKnownLocation(LocationManager.GPS_PROVIDER);                                                                                                                             
                               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~                                                                                                                              
  /home/daniel/Coding/noeppi-noeppi/nodecg-io-android/app/src/main/java/io/github/noeppi_noeppi/nodecg_io_android/Actions.java:291: Error: Call requires permission which may be rejected by user: code should explicitly check to see if permission is available (with checkPermission) or explicitly handle a potential SecurityException [MissingPermission]
          mgr.requestLocationUpdates(LocationManager.GPS_PROVIDER, time, distance, listener);                                                                                                                      
          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~                                                                                                                       
  /home/daniel/Coding/noeppi-noeppi/nodecg-io-android/app/src/main/java/io/github/noeppi_noeppi/nodecg_io_android/Actions.java:420: Error: Call requires permission which may be rejected by user: code should explicitly check to see if permission is available (with checkPermission) or explicitly handle a potential SecurityException [MissingPermission]
          String number = mgr.getLine1Number();                                                          
                          ~~~~~~~~~~~~~~~~~~~~                                                           
                                                    


* Try:                                              
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org                                                               

BUILD FAILED in 24s
```
So you it should say `./gradlew assemble` here (which worked for me) or you should fix the lint errors.

File: docs/samples/android.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+4. Launch the app. You'll be asked to grant the system alert window permission. This is required because the app will do its work in the background. But occasionally when user interaction is required, it needs to launch an activity without the user pressing the app icon in the launcher.
+
+5. Install the android developer tools and make sure you have the `adb` command in your `PATH`
+
+6. Enable developer options on you device and enable USB debugging there. See [here](https://developer.android.com/studio/debug/dev-options)
+
+7. Run `adb start-server`
+
+8. Connect your device via USB. You'll be prompted whetehr you want to allow USB debugging. Press `Allow`
```
Comment (daniel0611) [inline docs/samples/android.md:19]:
```suggestion
8. Connect your device via USB. You'll be prompted whether you want to allow USB debugging. Press `Allow`
```
Small typo here :wink:

**Review by daniel0611 — COMMENTED** [review]
It is working for me and looking good so far :smile:. I'll take a closer look and the service and app code tomorrow.

File: docs/samples/android.md (hunk 1 above)
Comment (noeppi-noeppi) [reply docs/samples/android.md:7]: lint should be fixed now

**Review by daniel0611 — APPROVED** [review]

**Pull request merged**